scipy/cluster/hierarchy.py

```diff
@@ -4233,9 +4232,9 @@ def leaders(Z, T):
 
     n_obs = Z.shape[0] + 1
 
-    # @wraps causes the Dask key to have prefix "leaders-*"
-    @wraps(_hierarchy.leaders)
-    def func(Z, T, validate):
+    # The function name is prominently visible on the user-facing Dask dashboard;
+    # make sure it is meaningful.
```

Comment:
Would it make sense to keep this comment in the source code here once for explanatory purposes, but rather than repeat it with every `lazy_apply` invocation, make it a prominent note in the [`lazy_apply` docs](https://data-apis.org/array-api-extra/generated/array_api_extra.lazy_apply.html)?

Reply:
https://github.com/data-apis/array-api-extra/pull/189